.github/scripts/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+##########################################################
+# GTSFM dependencies (including GTSAM) previously installed using conda
+##########################################################
+
+echo "Running .github/scripts/setup.sh..."
+conda init
+conda info --envs
+
+wget -O 2021_03_12_gtsam_python38_wheel.zip --no-check-certificate "https://drive.google.com/uc?export=download&id=1JvsLYngMzSqvmD6RwEgEb_HXXHfw_EJ2"
```

Comment:
this seems redundant with the code in `.github/scripts/python.sh`. Can we put it in just one place?

Reply:
This file is now removed.